### PR TITLE
I added a readme change so that we can have a cool time

### PR DESCRIPTION
**Pull request by sfanetti**
For story #885858585

### CODE REVIEW

**Review by gonzalezbri — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# sql-collab-project
```
Comment (ARod88) [inline README.md:1]: why did you put this change?

Comment (rdedwards1124) [inline README.md:1]: whats this?

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
```
Comment (rdedwards1124) [inline README.md:17]: change this to something

Comment (ARod88) [inline README.md:17]: use this instead 
## npm run

File: README.md [hunk 3]
```diff
@@ -1 +1,70 @@
...
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```
Comment (rdedwards1124) [inline README.md:9]: 9 is my favorite number! :)

**Review by ARod88 — CHANGES_REQUESTED** [review]

**Review by rdedwards1124 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sfanetti) [reply README.md:1]: I like this pattern

Comment (sfanetti) [reply README.md:1]: This is a baked potato

File: README.md (hunk 2 above)
Comment (sfanetti) [reply README.md:17]: I refuse make these changes because I refuse

File: README.md (hunk 3 above)
Comment (sfanetti) [reply README.md:9]: Well nein is mine

**Review by rdedwards1124 — APPROVED** [review]

**Review by ARod88 — APPROVED** [review]

**Pull request merged**